burst/README.md

```diff
@@ -0,0 +1,87 @@
+# Burst Metrics Service
+
+Burst metrics service is responsible for reading HPA metrics from a Kubernetes(k8s) API and return a formatted burst metrics header. It is implemented as a REST service.
+
+Currently it will capture current pod count and target pod count from an HPA.
+
```

Comment:
Currently implies it will do something different in the future - something like: The service captures ... seems more accurate

Reply:
Updated